setup/setup.sql

```diff
@@ -25,6 +25,10 @@ CREATE TABLE IF NOT EXISTS codes (
     code varchar(255)
 );
 
+CREATE TABLE IF NOT EXISTS version (
+    ver varchar(8)
+);
+
```

Comment:
It may be a good idea to add a timestamp to keep track of when the database has been updated.